src/Tizen.NUI/src/internal/Disposable.cs

```diff
@@ -27,7 +29,10 @@ public class Disposable : global::System.IDisposable
         /// <summary>
         /// A Flag to check if it is already disposed.
         /// </summary>
-        /// <since_tizen> 6 </since_tizen>
+        /// <since_tizen> 6 </since_tizen> 
+        /// When deleting the field, change it to private.
+        [Obsolete("Deprecated in API9, Will be removed in API11. Please use Disposed instead!")]
```

Comment:
Hello, 
I can see this is for Deprecated-ACR. 
please follow the steps below.
1) add comments like : `//ToDo : will raise deprecated-ACR, [Obsolete("Deprecated in API9, Will be removed in API11. Please use Disposed instead!")]`
2) get review in DevelNUI with No "ACR Required" tag
3) when it is merged in master branch by integration work, you can modify the comment of the [Obsolete] tag to be activated and use it for ACR process
4) the PR for ACR should be reviewed on master branch only

thank you.